lib/matplotlib/_constrained_layout.py

```diff
@@ -314,52 +295,77 @@ def do_constrained_layout(fig, renderer, h_pad, w_pad,
                         and ax._layoutbox is not None):
                     if ax.get_subplotspec().get_gridspec() == gs:
                         axs += [ax]
-            for ax in axs:
-                axs = axs[1:]
+            rownummin = 0*np.zeros(len(axs), dtype=np.int8)
+            rownummax = 0*np.zeros(len(axs), dtype=np.int8)
+            colnummin = 0*np.zeros(len(axs), dtype=np.int8)
+            colnummax = 0*np.zeros(len(axs), dtype=np.int8)
+            width = 0*np.zeros(len(axs))
+            height = 0*np.zeros(len(axs))
```

Comment:
Why are you multiplying all these "zeros" arrays by zero?

Reply:
:sheep:   Because I was multiplying by the axs array first but that didn’t work and then forgot to remove.  I’ll fix when I get home.